worker/src/test/java/gov/cms/ab2d/worker/TestUtil.java

```diff
@@ -0,0 +1,13 @@
+package gov.cms.ab2d.worker;
+
+import gov.cms.ab2d.common.util.FilterOutByDate;
+
+public final class TestUtil {
```

Comment:
What a horrid name for a class.

Unfortunately, the pattern exists elsewhere but is mitigated by being limited to the test arena.

Reply:
No need to fix this one.